x-pack/plugins/metrics_entities/server/modules/host_entities/host_entities_mapping.json

```diff
@@ -0,0 +1,45 @@
+{
+  "mappings": {
+    "_meta": {
+      "index": "host_ent"
+    },
+    "dynamic": "strict",
```

Comment:
for metrics usecases like this, think about disabling `_source`, this saves a lot of space and usually you don't need `_source` for transform cases like this.

Reply:
Like so below?
https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping-source-field.html#disable-source-field

In the security_solutions application there are a lot of spots that probably expect `_source` to be there. I will have to double check it. I might be ok to remove it before pushing up, I just have to check a few things first.

Thanks for the advice.